circle.yml

```diff
@@ -1,75 +0,0 @@
-# These environment variables must be set in CircleCI UI
-#
-# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
-# DOCKER_EMAIL   - login info for docker hub
-# DOCKER_USER
-# DOCKER_PASS
-#
-machine:
-  services:
-    - docker
-
-dependencies:
-  # make sure to keep the docker cache dir
-  cache_directories:
-    - "~/docker"
-
-  override:
-    - docker info
-
-    # build the container, use circleci's docker cache workaround
-    # only use 1 image per day to keep the cache size from getting 
-    # too big and slowing down the build
-    - I="image-$(date +%j).tar"; if [[ -e ~/docker/$I ]]; then echo "Loading $I"; docker load -i ~/docker/$I; fi
-
-    # create version.json
-    - >
-        printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' 
-        "$CIRCLE_SHA1" 
-        "$CIRCLE_TAG" 
-        "$CIRCLE_PROJECT_USERNAME" 
-        "$CIRCLE_PROJECT_REPONAME" 
-        "$CIRCLE_BUILD_URL"
-        > version.json
-    - cp version.json $CIRCLE_ARTIFACTS
-
-    - docker build -t syncserver:build .
-
-    - >
-        docker images --no-trunc | 
-        awk '/^app/ {print $3}' | 
```

Comment:
This was previously broken, as the image name is `syncserver` and not `app`, so nothing is awked and the shasum was empty. It's fixed now.